src/bin/flamegraph.rs

```diff
@@ -393,6 +402,7 @@ mod tests {
             bgcolors: Some(color::BackgroundColor::Blue),
             hash: true,
             palette_map: Default::default(),
+            #[cfg(feature = "nameattr")]
```

Comment:
This seems weird? Why is this part of this PR?

Reply:
This attribute `func_frameattrs` is conditional under `nameattr ` feature. Rust analyser shows this place as an issue and I confirm it. Ideally this case should be tested under CI as building and testing under different feature flags, but your pipeline is not customisable.

We can move this change into separate PR. I created 2 additional commits: clippy fix and this change, but supposed they are too small to place into separate PR.

Off-topic: do you want to move from azure pipelines to github actions?

Reply:
Ah, that's awkward.. That is not an okay `#[cfg]`, since `Options` and all of its fields are `pub`. Cargo features _must_ be additive, which this feature then isn't. We'll need to add `#[non_exhaustive]` to `Options` then, or remove the `#[cfg]` on that field. That's probably a priority to fix, and should happen in a separate PR.

We can customize the CI pipeline pretty easily. See for example
https://github.com/jonhoo/flurry/blob/f623ac994f20d35861799d7f106178a875f6e661/azure-pipelines.yml#L5 . I don't particularly want to move to GitHub actions, though I don't feel too strongly either way.

As for how to test this in CI, [`cargo-hack`](https://github.com/taiki-e/cargo-hack) would be the way to go.

Reply:
I'm handling this in https://github.com/jonhoo/inferno/pull/181.